### ISSUE TITLE
TabControl label alignment bug

### ISSUE THREAD

**Issue by Oylex** [UXDivers/Grial-UI-Kit-Support#413]
There is a bug with the TabControl / TabItem, when you navigate away and come back, the label is aligned to the left instead of the middle, I attached a gif:

![2018-04-18_11-37-51](https://user-images.githubusercontent.com/1819763/38942828-da33162a-42fd-11e8-951b-07e04983386a.gif)

By following the styles chain, the labels end up having these properties:

```
<Setter Property="HorizontalTextAlignment" Value="Center" />
<Setter Property="VerticalTextAlignment" Value="Center" />
```

I can't see why it changes, it seems to do it only once, and when I change tab it re-aligns itself and then if I do the procedure from the gif again it stays centered.

**Comment by jpbrocca** [comment]
Hi @Oylex,

can you please let me know:

- Is this happening with a real device or an emulator or both? Which device model are you using to test?
- which version of Xamarin Forms are you using?
- which version of Grial are you using?

Regards and thanks,
JP

**Comment by Oylex** [comment]
- I tested on my Galaxy S5 with Android 6.0.1 and it works, the problem occurs in my emulator which is a Nexus 5X running Android 8.0, I don't have access to a real device with a higher version than 6.0.1, so I can't tell if it's because of the emulator or the android version. It seems to work also on iPhone 8 iOS 11.3 emulator. I just tested on another emulator running Android 7.1 (Device: None, I think it was the default emulator) and I see the bug there as well.
- Xamarin Forms: 2.5.0.121934
- Grial: 2.6.4

If it happens only on the emulators then I don't mind...
